HumanRsourceManagementSystem/src/com/HRM/Adminusecases/LoginAdminUseCase.java

```diff
@@ -0,0 +1,47 @@
+package com.HRM.Adminusecases;
+
+import java.util.Scanner;
+
+import com.HRM.dao.AdminDao;
+import com.HRM.dao.AdminDaoImpl;
+import com.HRM.dto.AdminDTO;
+import com.HRM.exception.AdminException;
+
+public class LoginAdminUseCase {

```

Comment:
plesae give menaingful naming, this name looks more for a problem statement.
here just LOginAdmin should. be fine